src/eme.js

```diff
@@ -108,8 +108,9 @@ export const makeNewRequest = (requestOptions) => {
 
   eventBus.trigger('keysessioncreated');
 
-  return new Promise((resolve, reject) => {
+  player.on('dispose', keySession.close);
```

Comment:
does this work properly to have the `close()` method not directly attached/bound to `keySession`?
would it be better to bind it or wrap in an arrow function?